Inheritance deliverance mission(s) (#7327)
endless-sky/endless-sky: data/human/human missions.txt

			choice
				`	"I'll take the job."`
				`	"I don't like the idea of carrying an unknown cargo. Count me out."`
					goto nope
			label agree
			`	The man smiles. "I'm glad to hear that. We'll have the cargo loaded on your ship by sundown." He walks away into the spaceport, leaving you to prepare for the journey.`
				accept
			label nope
			`	The man frowns. "I see. Well, I appreciate that you listened to our offer, and I hope that you may assist us at a later date. Farewell." He walks away into the spaceport but doesn't appear to seek out any other captains.`
				decline
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		fail "Deal Change 2"
		payment 100000
		conversation
			`You step off your ship and order the spaceport crew to unload your cargo. As you do, a large, fat-lipped woman walks up to you and says in the most exaggerated manner possible, "Oh, thank you so much, captain! You must have gone through so much trouble to help little old me!" She gasps. "Oh, silly old me! I forgot to tell you who I am! I'm the recipient of the inheritance from <origin>, of course! You have truly made a great contribution to the galaxy!" She chuckles.`
			choice

> 				`	"Thank you?"`

Author: Yes it is.

				`	"Could you tell me what the inheritance is?"`
			`	Before you can get a single word out, the woman exclaims, "I'll be sure to inform your employers of the wonderful work you've done. Ta ta!" She begins to swagger towards a cream-colored car as her inheritance is loaded into a truck beside it. When you return to your ship's cockpit, you find that <payment> has been deposited into your account.`

mission "Deal Change 2"
	landing

> 	clearance `You read out the code on the back of the paper you received and receive a green light from Greenrock's authorities, along with a set of coordinates.`

Author: ```suggestion
	clearance `You read out the code on the back of the paper you received and get a green light from Greenrock's authorities, along with a set of coordinates.`
```

			label end
			`	You say your goodbyes, leaving to your ship and returning to the stars.`
				decline

mission "Deal Change 1"
	minor
	name "Inheritance to <planet>"
	description "Deliver <cargo> to <destination>. Payment will be <payment>."
	cargo "inherited assets" 5
	source
		attributes "paradise"
	destination "Skymoot"
	to offer
		random < 35
	on offer
		conversation

> 			`The spaceport on <origin> is as spirited as ever: to your left, a family wearing floral shirts pull along their luggage, with the man of the family flaunting a souvenir from some far away planet. To your right, a cleaner whistles a tuneless song while they wipe down a bench.`

Reviewer: ```suggestion
			`The spaceport on <origin> is as spirited as ever: to your left, a family wearing floral shirts pull along their luggage, with the man of the family flaunting a souvenir from some far-away planet. To your right, a cleaner whistles a tuneless song while they wipe down a bench.`
```

			`	A young man in a prim suit approaches you. "Good day, Captain! Would you mind if I spoke to you for a moment?"`
			choice
				`	"I'd be glad to listen."`
					goto listen
				`	"No, I'm busy right now."`
			`	The man frowns. "I see. Well, it's not like I was going to offer you a highly profitable job." He begins to shout while stepping away. "And that job certainly wouldn't have paid hundreds of thousands of credits!"`
			choice
				`	"Alright, what do you want?"`
					goto late
				`	"Shut the hell up."`
			`	Seemingly unfazed by your bluntness, the man cooly walks away into the rest of the spaceport while saying, "I'm sorry you feel that way captain. Perhaps you can assist us at a later date. Farewell."`
				decline
			label late
			`	He smirks as he walks back towards you. "I'm glad that you've changed your mind."`
			label listen

> 			`	The man pulls you aside to talk. "You see, one of our clients recently passed away, and in his will, he divided his property between each of his relatives. Only problem is that one of his family members, his niece, lives all the way down in the Rim. We would like you to deliver her portion of the inheritance, weighing <tons>, to her residence on <destination>. We can pay you <payment> upon completion of the job."`

Reviewer: ```suggestion
			`	The man pulls you aside to talk. "You see, one of our clients recently passed away, and in his will, he divided his property between each of his relatives. The only problem is that one of his family members, his niece, lives all the way down in the Rim. We would like you to deliver her portion of the inheritance, weighing <tons>, to her residence on <destination>. We can pay you <payment> upon completion of the job."`
```

			choice
				`	"I'll do it."`
					goto agree
				`	"What did she inherit?"`
				`	"Sorry, but I don't want to do this job."`
					goto nope
			`	"My sincerest apologies, but I'm afraid that is confidential information. Our prioritization of privacy is but one of our selling points."`
			choice
				`	"I'll take the job."`
				`	"I don't like the idea of carrying an unknown cargo. Count me out."`
					goto nope
			label agree
			`	The man smiles. "I'm glad to hear that. We'll have the cargo loaded on your ship by sundown." He walks away into the spaceport, leaving you to prepare for the journey.`
				accept
			label nope

> 			`	The man frowns. "I see. Well, I appreciate that you listened to our offer, and I hope that you may assist us at a later date. Farewell." He walks away into the spaceport, but he doesn't appear to seek out any other captains.`

Reviewer: ```suggestion
			`	The man frowns. "I see. Well, I appreciate that you listened to our offer, and I hope that you may assist us at a later date. Farewell." He walks away into the spaceport but doesn't appear to seek out any other captains.`
```

				decline
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		fail "Deal Change 2"
		payment 100000
		conversation
			`You step off your ship and order the spaceport crew to unload your cargo. As you do, a large, fat-lipped woman walks up to you and says in the most exaggerated manner possible, "Oh, thank you so much, captain! You must have gone through so much trouble to help little old me!" She gasps. "Oh, silly old me! I forgot to tell you who I am! I'm the recipient of the inheritance from <origin>, of course! You have truly made a great contribution to the galaxy!" She chuckles.`
			choice
				`	"Thank you?"`
				`	"Could you tell me what the inheritance is?"`
			`	Before you can get a single word out, the woman exclaims, "I'll be sure to inform your employers of the wonderful work you've done. Ta ta!" She begins to swagger towards a cream-colored car as her inheritance is loaded into a truck beside it. When you return to your ship's cockpit, you find that <payment> has been deposited into your account.`

mission "Deal Change 2"
	landing

> 	clearance `You read out the code on the back of paper you received, and receive a green light from Greenrock's authorities, along with a set of coordinates.`

Reviewer: ```suggestion
	clearance `You read out the code on the back of the paper you received and receive a green light from Greenrock's authorities, along with a set of coordinates.`
```

			``
			`<first> <last>,`
			`		You are currently carrying 5 tons of cargo to Skymoot. This cargo has been described to you as an "inheritance," and you have been promised 100 thousand credits as payment for the job. Under no circumstances should you allow the cargo to reach the destination. Instead, bring it to <destination>. You will be paid <payment> if you do so. The back of this note has a code you can use to pass through spaceport security.`
			``
			`	You flip the paper over and see a string of letters and numbers. The note has no signature, nor anything else to identify the person who wrote it.`
			choice
				`	(Keep the note.)`
					goto accept
				`	(Throw the note away.)`
			`	You stand up and crumple the note in your hand before throwing it into a bin. You think you catch a glance of a cloaked figure, but don't meet any more trouble on your way to the spaceport.`
				decline
			label accept
			`	You stand up and stuff the note in your pocket before continuing towards the spaceport.`
				accept

> 	to fail
> 		has "Deal Change 1: complete"

Author: ```suggestion
```

	on complete
		fail "Deal Change 1"
		payment 500000
		conversation
			`You follow the coordinates to a clearing within the forests of Greenrock. Several other ships are present near the landing site, all of which wear pirate colors. A person wearing a cloak steps out of one of the vessels to greet you. "I am glad that you were able to trust us," they say. "My name is not Ci, but you may refer to me as such." A gang of pirates emerges from the foliage around your ship and begin to unload the cargo from your ship.`
			choice
				`	(Stay silent.)`
					goto skip
				`	"Can I ask you some questions?"`

> 			`	One of the pirates near to you bares his pointy teeth, but Ci steps forward and raises their hand. "Please forgive the naivity of this captain," they say to the pirate before turning towards you. "I apologize, but I am afraid that I cannot answer your questions. This is a very volatile matter, and any misplaced information could spell death. You are safer knowing as little as possible."`

Reviewer: ```suggestion
			`	One of the pirates near you bares his pointy teeth, but Ci steps forward and raises their hand. "Please forgive the naivety of this captain," they say to the pirate before turning towards you. "I apologize, but I am afraid that I cannot answer your questions. This is a very volatile matter, and any misplaced information could spell death. You are safer knowing as little as possible."`
```

			choice
				`	"I understand."`
					goto skip
				`	"I'm taking back the cargo if you're not answering me."`
			`	Ci gestures towards the pirates around them. "Are you sure that is a wise choice, Captain?" They step closer. "The only reason you are still alive is because it is more convenient than killing you. It would be in your best interests to keep it that way."`
			choice
				`	"Okay, I'll stay quiet."`
					goto skip
				`	(Pull out your gun.)`
			`	The moment your hand touches the handle of your gun, you are blinded by bright lights and searing pain. You die before your body falls to the ground.`
				die
			label skip

> 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`

Reviewer: ```suggestion
			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
```

mission "Deal Change 1"
	minor
	name "Inheritance to <planet>"
	description "Deliver <cargo> to <destination>. Payment will be <payment>."
	cargo "inherited assets" 5
	source
		attributes "paradise"
	destination "Skymoot"
	to offer
		random < 35
	on offer
		conversation
			`The spaceport on <origin> is as spirited as ever: to your left, a family wearing floral shirts pull along their luggage, with the man of the family flaunting a souvenir from some far-away planet. To your right, a cleaner whistles a tuneless song while they wipe down a bench.`
			`	A young man in a prim suit approaches you. "Good day, Captain! Would you mind if I spoke to you for a moment?"`
			choice
				`	"I'd be glad to listen."`
					goto listen
				`	"No, I'm busy right now."`
			`	The man frowns. "I see. Well, it's not like I was going to offer you a highly profitable job." He begins to shout while stepping away. "And that job certainly wouldn't have paid hundreds of thousands of credits!"`
			choice
				`	"Alright, what do you want?"`
					goto late
				`	"Shut the hell up."`

> 			`	Seemingly unfazed by your bluntness, the man coolly walks away while saying, "I'm sorry you feel that way, captain. Perhaps you can assist us at a later date. Farewell."`

Reviewer 2: ```suggestion
			`	Seemingly unfazed by your bluntness, the man coolly walks away while saying, "I'm sorry you feel that way, Captain. Perhaps you can assist us at a later date. Farewell."`
```

			choice
				`	"I'll do it."`
					goto agree
				`	"What did she inherit?"`
				`	"Sorry, but I don't want to do this job."`
					goto nope
			`	"My sincerest apologies, but I'm afraid that is confidential information. Our prioritization of privacy is one of our main selling points."`
			choice
				`	"I'll take the job."`
				`	"I don't like the idea of carrying an unknown cargo. Count me out."`
					goto nope
			label agree
			`	The man smiles. "I'm glad to hear that. We'll have the cargo loaded onto your ship by sundown." He walks away into the spaceport, leaving you to prepare for the journey.`
				accept
			label nope
			`	The man frowns. "I see. Well, I appreciate that you listened to our offer, and I hope that you may assist us at a later date. Farewell." He walks away into the spaceport but doesn't appear to seek out any other captains.`
				decline
	on visit
		dialog phrase "generic cargo on visit"
	on complete
		fail "Deal Change 2"
		payment 100000
		conversation

> 			`You step off your ship and order the spaceport crew to unload your cargo. As you do, a large, fat-lipped woman walks up to you and says in the most exaggerated manner possible, "Oh, thank you so much, captain! You must have gone through so much trouble to help little old me!" She gasps. "Oh, silly old me! I forgot to tell you who I am! I'm the recipient of the inheritance from <origin>, of course! You have truly made a great contribution to the galaxy!" She chuckles.`

Reviewer 2: ```suggestion
			`You step off your ship and order the spaceport crew to unload your cargo. As you do, a large, fat-lipped woman walks up to you and says in the most exaggerated manner possible, "Oh, thank you so much, Captain! You must have gone through so much trouble to help little old me!" She gasps. "Oh, silly old me! I forgot to tell you who I am! I'm the recipient of the inheritance from <origin>, of course! You have truly made a great contribution to the galaxy!" She chuckles.`
```

			choice
				`	"Thank you?"`
				`	"Could you tell me what the inheritance is?"`
			`	Before you can get a single word out, the woman exclaims, "I'll be sure to inform your employers of the wonderful work you've done. Ta ta!" She begins to swagger towards a cream-colored car as her inheritance is loaded into a truck beside it. When you return to your ship's cockpit, you find that <payment> have been deposited into your account.`

mission "Deal Change 2"
	landing

> 	clearance `You read out the code on the back of the paper you received and get a green light from Greenrock's authorities, along with a set of coordinates.`
> 	name "Inheritance to <planet>"
> 	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so."
> 	source
> 		attributes "dirt belt" "south" "rim"
> 		not planet "Greenrock"
> 	destination "Greenrock"

Reviewer 2: ```suggestion
	name "Inheritance to <planet>"
	description "Instead of bringing the inheritance to Skymoot, deliver it to <destination>. You were promised <payment> for doing so."
	source
		attributes "dirt belt" "south" "rim"
		not planet "Greenrock"
	destination "Greenrock"
	clearance `You read out the code on the back of the paper you received and get a green light from Greenrock's authorities, along with a set of coordinates.`
```

	to offer
		has "Deal Change 1: active"
	on offer
		conversation
			`You touch down on <origin>'s surface and step out into the open air. The landing docks are emptier than usual, with only a handful of people walking around. Suddenly, you feel the wind knocked out of you as you collide with another person and fall on the pavement with a thud.`
			choice
				`	(Get up and apologize.)`
				`	(Get up and demand the other person's apology.)`
			`	You open your mouth as you lift yourself up, only to find that the other person is already gone. As you look around, you spot a piece of paper with writing lying on the ground beside you. You pick up the note and read it:`
			``
			`<first> <last>,`

> 			`		You are currently carrying 5 tons of cargo to Skymoot. This cargo has been described to you as an "inheritance," and you have been promised 100 thousand credits as payment for the job. Under no circumstances should you allow the cargo to reach the destination. Instead, bring it to <destination>. You will be paid <payment> if you do so. The back of this note has a code you can use to pass through spaceport security.`

Reviewer 2: ```suggestion
			`		You are currently carrying five tons of cargo to Skymoot. This cargo has been described to you as an "inheritance," and you have been promised 100,000 credits as payment for the job. Under no circumstances should you allow the cargo to reach the destination. Instead, bring it to <destination>. You will be paid <payment> if you do so. The back of this note has a code you can use to pass through spaceport security.`
```

			choice
				`	(Stay silent.)`
					goto skip
				`	"Can I ask you some questions?"`
			`	One of the pirates near you bares his pointy teeth, but Ci steps forward and raises their hand. "Please forgive the naivety of this captain," they say to the pirate before turning towards you. "I apologize, but I am afraid that I cannot answer your questions. This is a very volatile matter, and any misplaced information could spell death. You are safer knowing as little as possible."`
			choice
				`	"I understand."`
					goto skip
				`	"I'm taking back the cargo if you're not answering me."`
			`	Ci gestures towards the pirates around them. "Are you sure that is a wise choice, Captain?" They step closer. "The only reason you are still alive is because it is more convenient than killing you. It would be in your best interests to keep it that way."`
			choice
				`	"Okay, I'll stay quiet."`
					goto skip

> 				`	(Pull out your gun.)`

Reviewer 2: ```suggestion
				`	(Pull out my gun.)`
```
